frontend/src/metabase/visualizations/components/ChartSettingsWidgetPopover.tsx

```diff
@@ -27,7 +21,7 @@ const ChartSettingsWidgetPopover = ({
   const contentRef = useRef<HTMLDivElement>(null);
 
   useEffect(() => {
-    sections.current = _.chain(widgets).pluck("section").unique().value();
+    sections.current = [...new Set(widgets.map(widget => widget.section))];
```

Comment:
changed this because underscore was being weird about types

Reply:
```suggestion
    sections.current = Array.from(new Set(widgets.map(widget => widget.section)));
```